app/src/main/java/nextstep/signup/ui/component/InputView.kt

```diff
@@ -0,0 +1,45 @@
+package nextstep.signup.ui.component
+
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun InputView(hint: String, changeListener: (String) -> Unit) {
+    val inputValue by remember { mutableStateOf("") }
+    TextField(modifier = Modifier
+        .width(296.dp)
```

Comment:
가로 화면에 꽉 차게 구현하고 싶을 때에는 다음 수정자를 활용하시면 됩니다!
```suggestion
        .fillMaxWidth()
```
fillMaxHeight, fillMaxSize 등도 제공합니다.